certified-connectors/Adobe Experience Manager/apiDefinition.swagger.json

```diff
@@ -0,0 +1,431 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Adobe Experience Manager",
+    "description": "Adobe Experience Manager APIs allow access to Adobe Experience Manager Forms related services(e.g. Communications and more). This integration will also allow flows to be triggered when a form is submitted through Adobe Experience Manager Forms.",
+    "version": "1.0.0",
+    "contact": {
+      "name": "Adobe Experience Manager Forms",
+      "url": "https://www.adobe.com/go/adobe-experience-manager-forms-community"
```

Comment:
@syamavarm please add "email": "Your Email" to the contact section.

Reply:
Added the email.